pkgs/applications/misc/tabula-java/default.nix

```diff
@@ -1,24 +1,74 @@
-{ stdenv, lib, fetchurl, jre, makeWrapper }:
+{ fetchFromGitHub
+, jre
+, lib
+, makeWrapper
+, maven
+, stdenv
+}:
 
-stdenv.mkDerivation rec {
+let
   pname = "tabula-java";
   version = "1.0.5";
-
-  src = fetchurl {
-    url = "https://github.com/tabulapdf/tabula-java/releases/download/v${version}/tabula-${version}-jar-with-dependencies.jar";
-    sha256 = "sha256-IWHj//ZZOdfOCBJHnPnKNoYNtWl/f8H6ARYe1AkqB0U=";
+  src = fetchFromGitHub {
+    owner = "tabulapdf";
+    repo = "tabula-java";
+    rev = "v${version}";
+    hash = "sha256-lg8/diyGhfkUU0w7PEOlxb1WNpJZVDDllxMMsTIU/Cw=";
   };
+  fetchedMavenDeps = stdenv.mkDerivation ({
```

Comment:
Can all these be converted to `buildMavenPackage`?

Reply:
Thanks, done!